### ISSUE TITLE
force.js error in some demos

### ISSUE THREAD

**Issue by freedigin** [BeepIsla/CSGO-Overwatch-Bot#38]
i use force.js to analyzed demo from my retakes server but it didn't work in some demos (error)

lastest version

```
E:\Demo CSGO\CSGO-Overwatch-Bot-master\node_modules\demofile\dist\entities\player.js:85
    return this.userInfo.guid;
                         ^

TypeError: Cannot read property 'guid' of null
    at Player.get steamId [as steamId] (E:\Demo CSGO\CSGO-Overwatch-Bot-master\node_modules\demofile\dist\entities\player.js:85:26)
    at E:\Demo CSGO\CSGO-Overwatch-Bot-master\force.js:71:46
    at Array.map (<anonymous>)
    at GameEvents.getPlayerIndex (E:\Demo CSGO\CSGO-Overwatch-Bot-master\force.js:71:35)
    at GameEvents.emit (events.js:223:5)
    at E:\Demo CSGO\CSGO-Overwatch-Bot-master\node_modules\demofile\dist\gameevents.js:30:22
    at Array.forEach (<anonymous>)
    at DemoFile.<anonymous> (E:\Demo CSGO\CSGO-Overwatch-Bot-master\node_modules\demofile\dist\gameevents.js:29:30)
    at DemoFile.emit (events.js:228:7)
    at DemoFile._parseRecurse (E:\Demo CSGO\CSGO-Overwatch-Bot-master\node_modules\demofile\dist\demo.js:269:12)
```

**Comment by BeepIsla** [comment]
Can you try updating demofile by using `npm install demofile` and seeing if that resolves the problem?

The demofile version used in this bot a little bit older and maybe that fixes it but I doubt it.

**Comment by freedigin** [comment]
> Can you try updating demofile by using `npm install demofile` and seeing if that resolves the problem?
> 
> The demofile version used in this bot a little bit older and maybe that fixes it but I doubt it.

not working i still have same errorr

**Comment by BeepIsla** [comment]
Can you share the demo this happens on?

**Comment by freedigin** [comment]
> Can you share the demo this happens on?

Sure 😃 

```
Download : https://drive.google.com/file/d/15YJhs2MOwu2O9x2IDN3lKEp6MiuktRJo/view?usp=sharing
Steam Suspect : 76561198446365476

The Suspect start playing on round 10
```

**Comment by BeepIsla** [comment]
Thanks for the bug report, I have fixed the issue (7b9d5199efa21a450b2107872717dc464ca1f825). Please update and try again.

**Comment by freedigin** [comment]
> Thanks for the bug report, I have fixed the issue ([7b9d519](https://github.com/BeepIsla/CSGO-Overwatch-Bot/commit/7b9d5199efa21a450b2107872717dc464ca1f825)). Please update and try again to confirm.

hi, thank you so much now it's work.
